### DISCUSSION TITLE
DatePicker COPY/PASTE

### DISCUSSION THREAD

**Discussion by 1lkin13** [heroui-inc/heroui#4714] [Feature requests]
### Is your feature request related to a problem? Please describe.

1)I cannot copy paste "datepicker" according to local time and select.  For example, copy and paste the date 12.05.2004 and let it appear as 12/05/2004.  At the same time, 12/05/2004 Let me select it by clicking with the mouse or in phone mode and copy it.
2)When we use form validation libraries such as Formik, the date is written as 17.05.2004 and the onchange time is written as 17.05.200. If we write 4 more, it says this is an error. He says it doesn't fit the pattern of history. Add date part minimum 2 maximum 3 or debounce

### Describe the solution you'd like

1)I cannot copy paste "datepicker" according to local time and select.  For example, copy and paste the date 12.05.2004 and let it appear as 12/05/2004.  At the same time, 12/05/2004 Let me select it by clicking with the mouse or in phone mode and copy it.
2)When we use form validation libraries such as Formik, the date is written as 17.05.2004 and the onchange time is written as 17.05.200. If we write 4 more, it says this is an error. He says it doesn't fit the pattern of history. Add date part minimum 2 maximum 3 or debounce

### Describe alternatives you've considered

1)I cannot copy paste "datepicker" according to local time and select.  For example, copy and paste the date 12.05.2004 and let it appear as 12/05/2004.  At the same time, 12/05/2004 Let me select it by clicking with the mouse or in phone mode and copy it.
2)When we use form validation libraries such as Formik, the date is written as 17.05.2004 and the onchange time is written as 17.05.200. If we write 4 more, it says this is an error. He says it doesn't fit the pattern of history. Add date part minimum 2 maximum 3 or debounce

### Screenshots or Videos

https://github.com/user-attachments/assets/4bff1c20-d65d-475f-9198-975fa9e6308d

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1666/datepicker-copypaste">ENG-1666 DatePicker COPY/PASTE</a></p>

**Comment by 1lkin13** [comment]
> [ENG-1666 DatePicker COPY/PASTE](https://linear.app/nextui-inc/issue/ENG-1666/datepicker-copypaste)

![Screenshot 2024-12-14 233520](https://github.com/user-attachments/assets/304a0849-b9d0-40f5-9a78-03f1b79c779e)
